source/usage-examples/command.txt

```diff
@@ -0,0 +1,49 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. A raw database operation is a
+command you can execute directly on the MongoDB server CLI. These
+commands include administrative and diagnostic tasks, such as fetching
+server stats or initializing a replica set. Call the ``runCommand()``
+method with a Bson command object on an instance of a ``MongoDatabase``
+to run your raw database operation.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java driver whenever possible.
```

Comment:
Suggestion: It might be good to include why?  They amount to the same thing, its just the mongo shell is often quicker and easier.

Reply:
Done, thanks for the suggestion! I imagine a lot of new users would question why, so this is definitely a good addition.